packages/flutter/lib/src/cupertino/segment_control.dart

```diff
@@ -0,0 +1,191 @@
+import 'package:flutter/widgets.dart';
+import 'package:flutter/cupertino.dart';
```

Comment:
You can't quite do this. It creates a dependency loop (see travis test failure). Import the specific sibling files you need. See other files in this directory.